driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -35,13 +35,13 @@
 import static com.datastax.driver.core.TestUtils.executeNoFail;
 import static com.datastax.driver.core.TestUtils.findAvailablePort;
 
-public class CCMBridge implements Closeable {
+public class CCMBridge implements CCMAccess {
 
     private static final Logger logger = LoggerFactory.getLogger(CCMBridge.class);
 
     private static final String CASSANDRA_VERSION;
 
-    private static final String CASSANDRA_INSTALL_ARGS;
+    private static final Set<String> CASSANDRA_INSTALL_ARGS;
```

Comment:
Changed to a set to be able to override some of the arguments (typically, the version).